adapters/Vungle/VungleAdapter/GADMAdapterVungleRewardedAd.m

```diff
@@ -192,4 +182,18 @@ - (void)adNotAvailable:(nonnull NSError *)error {
   [[GADMAdapterVungleRouter sharedInstance] removeDelegate:self];
 }
 
+- (void)trackClick {
+  id<GADMediationRewardedAdEventDelegate> strongDelegate = _delegate;
+  [strongDelegate reportClick];
+}
+
+- (void)rewardUser {
+  id<GADMediationRewardedAdEventDelegate> strongDelegate = _delegate;
+  [strongDelegate didEndVideo];
+  GADAdReward *reward =
+  [[GADAdReward alloc] initWithRewardType:@"vungle"
+                             rewardAmount:[NSDecimalNumber decimalNumberWithString:@"1"]];
+  [strongDelegate didRewardUserWithReward:reward];
+}
+
```

Comment:
We don't need `- (void)willLeaveApplication ` for `GADMAdapterVungleRewardedAd`? Just making sure.

Reply:
@ashinagawa
Hi Aki, because the protocol `GADMediationRewardedAdEventDelegate <GADMediationAdEventDelegate>`（https://developers.google.com/ad-manager/mobile-ads-sdk/ios/api/reference/Protocols/GADMediationRewardedAdEventDelegate）doesn't contain `-(void)willLeaveApplication` method to call. In the spreadsheet `Vungle Mediation Adapter Callback Event Table` that you summarized(https://docs.google.com/spreadsheets/d/1d0bXJ_1cTkFp3Zox54dqn0WyL0DTW_wvy_9knzcxq8o/edit#gid=0), you also didn't mark the `- (void)willLeaveApplication` needs to be implemented in `GADMAdapterVungleRewardedAd` class of AdMob Adapter. Thanks a lot. 🙂

Reply:
Got it!